contracts/src/v0.8/functions/dev/1_0_0/libraries/FunctionsResponse.sol

```diff
@@ -21,8 +21,8 @@ library FunctionsResponse {
   }
 
   enum FulfillResult {
-    USER_SUCCESS, // 0
-    USER_ERROR, // 1
+    USER_CALLBACK_SUCCESS, // 0
```

Comment:
How about just "SUCCESS"? This covers a little more than just callback, it's an e2e sucess.

Reply:
Not quite.  `USER_CALLBACK_SUCCESS` could still have just successfully sent error bytes to the client's callback.
A complete success is empty error bytes && `USER_CALLBACK_SUCCESS`

Reply:
I guess it depends how you look at it. It's an e2e processing success that can contain either a computation result or an error. Not sure what a good name fir that could be. Maybe "FULFILLED"? Or "COMPLETED"?

Reply:
I think USER_CALLBACK_SUCCESS is most descriptive and accurate, especially for any users reading our contracts.

Reply:
Can we still consider FULFILLED? I think it makes sense for a FulfillResult to be equal to FULFILLED?

Reply:
I see a merit in combining to USER_CALLBACK_FULFILLED

The word success can be easily assumed to mean the JS was successful, which means, when debugging people may not think about separating the callback from the execution of the JS.

Reply:
I'm +1 on what @bolekk and @zeuslawyer said. What do you think @KuphJr ?

Reply:
I don't like USER_CALLBACK_FULFILLED because we don't really fulfill a callback, we fulfill a request. Is the "USER_CALLBACK_" prefix adding any useful information?

Reply:
@bolekk that makes perfect sense from the POV of engineering technical succinctness. On the other hand, my concern is for end users who are not CL engineers.

Expectations when reading code are influenced by our knowledge of the code. For someone not intimate with the code, a reasonable expectation for "FULFILLED" with no other helpful info will be to expect SUCCESS.

It is our responsibility to convey that FULFILLED is the code WAI from *our* POV but may not be WAI from the end users POV.

Reply:
I agree it was true for SUCCESS but I'm not so sure about FULFILLED. It's not perfect but USED_CALLBACK_FULFILLED almost makes me pause and wonder if it's positive or negative ("wait, is user callback supposed to fulfill or not...?"). I think we also use the word "fulfill" in docs, comments and tooling messages to indicate what it is for people who bother to read.